Backend/app.py

```diff
@@ -91,7 +91,9 @@ def register():
     logger.info(str(flaskResponse))
     return flaskResponse
   dynamoDBInstance = getDatabaseClient()
-  res = user_ctrl.register(request.json, mail, dynamoDBInstance)
+  httpOrigin = request.environ.get('HTTP_ORIGIN')
+  jsonData = request.json
+  res = user_ctrl.register(httpOrigin, jsonData, mail, dynamoDBInstance)
```

Comment:
Why is the request.json being put in a variable here. No need to change, just curious.